core/src/voting_service.rs

```diff
@@ -78,12 +81,25 @@ impl VotingService {
             inc_new_counter_info!("tower_save-ms", measure.as_ms() as usize);
         }
 
-        let _ = cluster_info.send_transaction(
-            vote_op.tx(),
-            next_leader_tpu_vote(cluster_info, poh_recorder)
-                .map(|(_pubkey, target_addr)| target_addr),
+        // Attempt to send our vote transaction to the leaders for the next few slots
+        const UPCOMING_LEADER_FANOUT_SLOTS: u64 = FORWARD_TRANSACTIONS_TO_LEADER_AT_SLOT_OFFSET;
+        #[cfg(test)]
+        static_assertions::const_assert_eq!(UPCOMING_LEADER_FANOUT_SLOTS, 2);
```

Comment:
Is there data on why 2 is chosen versus any other number? Also I'm not sure re-using this makes sense. It seems like an offset not a boundary for forwarding.

Reply:
2 offset slots is the current behavior of forwarding. This patch minimally changes behavior by additionally sending to the leader at offset spot 1.

Of course we should look closer at forwarding in the future but this change is an incremental improvement on what we have already been doing